cmd/hosts/install.go

```diff
@@ -27,7 +30,7 @@ var installCmd = &cobra.Command{
 		}
 
 		buildConfig := artifact_builder.NewBuilderConfig().WithBootstrap(bootstrapConfig).WithHappyConfig(happyConfig)
-		containers, err := buildConfig.GetContainers()
+		containers, err := buildConfig.GetContainers(ctx)
```

Comment:
```suggestion
		containers, err := buildConfig.GetContainers(cmd.Context())
```
I think cmd has a ctx we should use but now don't remember